### DISCUSSION TITLE
Site down issue when running  enlightn in laravel

### DISCUSSION THREAD

**Discussion by pratikshap31** [enlightn/enlightn#160] [Q&A]
Hello,

I am trying to run the Enlightn package on my Laravel application using the following command:

```
php artisan enlightn
```

It is executing properly; however, it significantly slows down my application and sometimes even causes it to go down. After investigating, I noticed that Enlightn consumes a large amount of server memory during its execution. I checked the memory usage using the command `free -m`.

Could anyone kindly assist me with this issue?

**Comment by pratikshap31** [comment]
@paras-malhotra 

Can you please help me here?

**Reply by paras-malhotra** [reply]
I guess that means that you don't have enough server memory. There are only two solutions - either don't run Enlightn in production or jack up the production server memory.

**Reply by pratikshap31** [reply]
@paras-malhotra 

I have run it many times in production, but after installing the queue, it takes a long time to run enlightn and consumes all memory. As per my understanding, it is not merely a server memory issue.
